documentation/README.md

```diff
@@ -0,0 +1,71 @@
+[strimziDoc]: https://strimzi.io/documentation/ "Strimzi documentation"
+[strimziContrib]: https://strimzi.io/contributing/guide/ "Strimzi documentation contributor guide"
+
+<!-- omit from toc -->
+# Strimzi documentation
+
+Welcome to the Strimzi documentation! This repository contains the source files for managing the Strimzi documentation. The documentation is written in AsciiDoc, and it covers guides to help you understand, deploy, and configure Strimzi.
```

Comment:
Should it be `This folder ...`  rather than `This repository ...`? Since it is only one part of it.